### PR TITLE
Scheduler

### PR DESCRIPTION
**Pull request by chsclarke**
### Usage:
```python
scheduler = Scheduler()

# or minuets=1 or hours=1 ... etc. check documentation for other input args
id = scheduler.create_job(myfunc, seconds=1)

print(scheduler.list_jobs())

scheduler.remove_job(id)
```

Note that all job creations and deletions are logged in `/scheduler/scheduler.log`

All input args for `Scheduler.create_job(myfunc, args)` can be found [here](https://apscheduler.readthedocs.io/en/stable/modules/triggers/interval.html)

### CODE REVIEW

File: scheduler/scheduler.py [hunk 1]
```diff
@@ -6,6 +6,7 @@ class Scheduler():
     def __init__(self):
         self.sched = BackgroundScheduler(timezone='EST')
         self.jobSet = []
+        self.sched.start()
         logging.basicConfig(filename='scheduler.log',level=logging.DEBUG)
```
Comment (rjewing) [inline scheduler/scheduler.py]: It seems like scheduling jobs works, but you never grab any values or send it to the function. You should write a wrapper function for the scheduled task that grabs the newest values and passes them as arguments.
```
def task_wrapper(actual_func):
    patient_id, bps, bpd, pulse, oxygen = data_vendor.get_values()
    actual_func(patient_id, bps=bps, bpd=bpd, pulse=pulse, oxygen=oxygen)

```

Comment (rjewing) [reply scheduler/scheduler.py]: It doesn't have to be exactly this, but something similar should work pretty well.

**Review by rjewing — CHANGES_REQUESTED** [review]
Everything looks fine. You should just grab values from the data_vendor and pass those along. All the tasks will have the following function parameters:
```
def task(patient_id, oxygen=None, pulse=None, bps=None, bpd=None):
    # implementation here
```

**Review by YahiaBakour — CHANGES_REQUESTED** [review]
I'm not approving anythang untill you add docs 

thnx

**Comment by dlhi** [comment]
As far as I am aware, you will be grabbing the generated data from Ryan's code and then calling my CRUD update function to add the entry to the database . . .

The function to add a data entry using my db code is:

`add_all_into_data(patientID, oxygen=someval, pulse=someval, bps=someval, bpd=someval)`

Using parameterized functions, the order does not matter as long as you specify the parameter e.g.

`add_all_into_data('1141', bps='120', bpd='80', oxygen='0.97', pulse='100')`

All values passed into this function should be strings

For more documentation, take a look at my code under database/database.py

File: scheduler/scheduler.py (hunk 1 above)
Comment (chsclarke) [reply scheduler/scheduler.py]: added it

**Review by rjewing — APPROVED** [review]
LGTM! Although we should probably add the scheduler.log file to the .gitignore.

**Pull request merged**